src/setup/kmd/mod.rs

```diff
@@ -0,0 +1,6 @@
+//! Utilities for setting up and tearing down Key Management Daemon (kmd) instances.
+
+pub mod config;
+pub mod constants;
+#[allow(clippy::module_inception, dead_code)]
```

Comment:
Should I move the contents of the kmd.rs here to resolve `clippy::module_inception`?

Reply:
Done